controller/AdminAccessController.php

```diff
@@ -109,23 +113,16 @@ public function savePermissions(): void
         $recursive = ($this->getRequest()->getParameter('recursive') === '1');
 
         try {
-            $this->validateCsrf();
```

Comment:
Why, though?
Do we plan to open that endpoint to cross-site requests?

Reply:
New request handler works different way with csrf tokens. I have no solution how to implement here csrf tokens.